### DISCUSSION TITLE
sync across devices like it is done for internal windows clipboard history

### DISCUSSION THREAD

**Discussion by Mk0IV** [hluk/CopyQ#1699] [Ideas]
Hi!

I think it would be nice to sync clipboard history between devices, like it is done with win+V history in windows. Perhaps using tab, google drive/one drive. For sure with encryption, may be using different tabs with device names? What do you say. Android version/access would be cool too :)

Br,
Mike

**Comment by hluk** [comment]
You can synchronize tabs in CopyQ to shared folders on disk: https://copyq.readthedocs.io/en/latest/synchronize.html

**Reply by hussain7college** [reply]
Thank you, and then he could sync it using any sync option (google drive, one drive, rsync, github with crontab)

**Reply by abdullahalkheshen** [reply]
Would you please explain how in details?
